### PR TITLE
Add missing Kind to sql agg assignment

### PR DESCRIPTION
**Pull request by mattnibs**
Closes brimdata/brim#1884

### CODE REVIEW

File: lake/ztests/query-sql-agg.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# brimdata/brim#1884
+
+script: |
+  export ZED_LAKE_ROOT=test
+  zed lake init -q
+  zed lake create -q test
+  zed lake use -q test@main
+  zed lake load -q data.zson
+  zed lake query -z 'from test | SELECT count(), id GROUP BY id | sort id'
```
Comment (nwt) [inline lake/ztests/query-sql-agg.yaml:9]: No need for `from` after `zed lake use`.
```suggestion
  zed lake query -z 'SELECT count(), id GROUP BY id | sort id'
```

File: service/ztests/query-sql-agg.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# brimdata/brim#1884
+
+script: |
+  source service.sh
+  zed lake create -q test
+  zed lake use -q test@main
+  zed lake load -q data.zson
+  zed lake query -z 'from test | SELECT count(), id GROUP BY id | sort id'
```
Comment (nwt) [inline service/ztests/query-sql-agg.yaml:8]:
```suggestion
  zapi create -q test
  zapi use -q test@main
  zapi load -q data.zson
  zapi query -z 'SELECT count(), id GROUP BY id | sort id'
```

Comment (mattnibs) [reply service/ztests/query-sql-agg.yaml:8]: I'd prefer to keep using `zed lake` since if I make a change I can easily copy this to lake and vice versa.

Comment (nwt) [reply service/ztests/query-sql-agg.yaml]: But this needs to use `zapi` or `zed api` in order to talk to `zed lake serve`. `zed lake` just goes straight to the file system.

Comment (mccanne) [reply service/ztests/query-sql-agg.yaml]: Good catch!  I missed that one.

**Review by nwt — APPROVED** [review]

**Pull request merged**